input/includes/iti-mhd-fxca-update.html

```diff
@@ -0,0 +1,17 @@
+<div class="mermaid">
+%%{
+    init: {
+        'theme': 'neutral',
+        'themeVariables': {
+            'fontSize': '14px'
+        }
+    }
+}%%
+flowchart LR
+    A[Document Source] -->|Update Doc Metadata| B(DocumentRecipient
+    Opt FXCA
+    Grouped with Document Source)
+    B --> C(own Community?)
+    C -->|yes| B 
```

Comment:
That probably makes an infinite loop. It should be similar to the retrieve diagram, but is not (different shape, different flows).